Core/include/Acts/EventData/VectorMultiTrajectory.hpp

```diff
@@ -16,7 +16,9 @@ namespace Acts {
 
 class VectorMultiTrajectory final
     : public MultiTrajectory<VectorMultiTrajectory> {
+#ifndef DOXYGEN
   friend MultiTrajectory<VectorMultiTrajectory>;
+#endif
```

Comment:
out of curiosity: why does doxgen complain about `friend`?

Reply:
🤷  Doxygen is in fact fine, but sphinx+breathe somehow get tripped up. I could actually remove this to check if this also goes away with Doxygen 1.9.4, I didn't try that exact combination.